features/works/work_download.feature

```diff
@@ -109,8 +109,8 @@ Feature: Download a work
     And "Published:" should appear before "Completed"
     And "Completed:" should appear before "Chapters"
     # note difference with the work: in the work "Words" come before "Chapters"
-    And "Chapters:" should appear before "Words"
-    And "Words:" should appear before "Could be downloaded"
+    And "Words:" should appear before "Chapters:"
```

Comment:
Can you remove the now-inaccurate comment above this, and the one on line 97 as well?